src/main/java/edu/asu/commons/foraging/conf/ServerConfiguration.java

```diff
@@ -29,11 +29,11 @@
 
     private static final long serialVersionUID = -1737412253553943902L;
 
-    private static final double DEFAULT_SHOW_UP_PAYMENT = 5.0d;
-    private static final double DEFAULT_QUIZ_CORRECT_ANSWER_REWARD = 0.50d;
+    private static final double DEFAULT_SHOW_UP_PAYMENT = 50.0d;
```

Comment:
This can be configured in the specific configuration files (`server.xml / roundN.xml`) instead since we are still planning experiments to be run in Louisville where $50.00 is quite a bit for a show up payment..